.travis.yml

```diff
@@ -0,0 +1,27 @@
+language: node_js
+services:
+  - docker
+#   Run all unit tests under the following environments
+#   `npm test`
+stage: "unit tests"
+node_js:
+  - "12.13.0"
```

Comment:
Should we add `latest` (I think that's a thing you can do), too?

Reply:
Thought about it but `latest` is a moving target meaning that the next person to pick up the repo might find tests failing.
Here we're specifying a contract that we're supporting `12.13.0`, upgrading should be an explicit process

Reply:
Not really, right? We should always be compatible with the latest stable release of Node.

Reply:
Hmmmmm, I'm a bit split, on the one hand the next person that picks up the repo, it might be completely broken, on the other it sorta forces them to address the breaking changes and keep us up to date, otherwise it'll never be upgraded, I don't really mind @jcolgan-r7 what you think

Reply:
The version is pulled in only as part of the initial build, right? In that case, I'd vote for adding `latest`.

Reply:
grand job